package.json

```diff
@@ -14,28 +14,25 @@
     "url": "git+https://github.com/adobe/leonardo.git"
   },
   "contributors": [
-    "Larry Davis <lazdnet@gmail.com>"
+    "Larry Davis <lazdnet@gmail.com>",
+    "Frank Karstens <karstens@adobe.com>"
   ],
   "license": "Apache-2.0",
   "devDependencies": {
     "@changesets/cli": "^2.27.1",
     "@commitlint/cli": "^18.4.3",
     "@commitlint/config-conventional": "^18.4.3",
-    "@moonrepo/cli": "^1.18.1",
+    "@moonrepo/cli": "^1.22.4",
     "@parcel/packager-raw-url": "2.11.0",
     "@parcel/transformer-sass": "2.11.0",
     "@parcel/transformer-webmanifest": "2.11.0",
     "ava": "^6.0.1",
-    "buffer": "^5.5.0||^6.0.0",
     "husky": "^8.0.3",
     "lint-staged": "^15.2.0",
-    "path-browserify": "^1.0.0",
-    "prettier": "^3.1.1",
-    "process": "^0.11.10",
-    "vm-browserify": "^1.1.2"
+    "prettier": "^3.1.1"
   },
   "engines": {
     "node": "20.0.0"
```

Comment:
@GarthDB I wonder, if the node version should be updated here or in the moon toolchain?

```
➜  leonardo git:(monorepo) pnpm install
 WARN  Unsupported engine: wanted: {"node":"20.0.0"} (current: {"node":"v20.11.1","pnpm":"8.15.3"})
```

Reply:
Probably just the moon config.